build.sh

```diff
@@ -217,7 +216,15 @@ case $OSName in
 
     Linux)
         __HostOS=Linux
-        __TestNugetRuntimeId=ubuntu.14.04-x64
+        source /etc/os-release
+        if [ $ID = "centos" ]; then
+            __TestNugetRuntimeId=centos.7.1-x64
```

Comment:
Can you change this to `centos.7-x64`?  That the RID that will be generated by default on CentOS platforms.  I would like to remove the centos.7.1-x64 rid at some point because it's meaningless.